packages/website/public/llms/styleGuide.md

```diff
@@ -0,0 +1,417 @@
+- Be careful with sigmoid functions. Sigmoid curves with distributions can have very little uncertainty in the middle, and very high uncertainty at the tails. If you are unsure about these values, consider using a mixture distribution. For example, this curve has very high certainty in the middle, and very high uncertainty at the tails: `adoption_rate(t: inputs.t) = 1 / (1 + exp(-normal(0.1, 0.08) * (t - 30)))`
+- Make sure to flag any variables that are highly speculative. Use @doc() to explain that the variable is speculative and to give a sense of the uncertainty. Explain your reasoning, but also warn the reader that the variable is speculative.
+
+### Percentages / Probabilities
+
+- Use a @format() tag, like ".0%" to format percentages.
+- If using a distribution, remember that it shouldn't go outside of 0% and 100%. You can use beta distributions or truncate() to keep values in the correct range.
+- If you do use a beta distribution, keep in mind that there's no ({p5, p95}) format. You can use beta(alpha:number, beta:number) or beta({mean: number, stdev: number}) to create a beta distribution.
+- Write percentages as "5%" instead of "0.05". It's more readable.
+
+### Domains
+
+- Prefer using domains to throwing errors, when trying to restrict a variable. For example, don't write, "if year < 2023 then throw("Year must be 2023 or later")". Instead, write f(t: [2023, 2050]).
+- Err on the side of using domains in cases where you are unsure about the bounds of a function, instead of using if/throw or other error handling methods.
+- If you only want to set a min or max value, use a domain with Number.maxValue or -Number.maxValue as the other bound.
+- Do not use a domain with a complete range, like [-Number.maxValue, Number.maxValue]. This is redundant. Instead, just leave out the domain, like "foo(f)".
+
+```squiggle
+// Do not use this
+f(t: [-Number.maxValue, Number.maxValue]) + 1
+
+// Do this
+f(t) = t + 1
+```
+
+## Structure and Naming Conventions
+
+### Structure
+
+- Don't have more than 10 variables in scope at any one time. Feel free to use many dictionaries and blocks in order to keep things organized. For example,
+
+```squiggle
+@name("Key Inputs")
+inputs = {
+  @name("Age (years)")
+  age = 34
+
+  @name("Hourly Wage ($/hr)")
+  hourly_wage = 100
+
+  @name("Coffee Price ($/cup)")
+  coffee_price = 1
+  {age, hourly_wage, health_value, coffee_price}
+}
+```
+
+Note: You cannot use tags within dicts like the following:
+
+```squiggle
+// This is not valid. Do not do this.
+inputs = {
+  @name("Age (years)")
+  age = 34,
+
+  @name("Hourly Wage ($/hr)")
+  hourly_wage: 100,
+}
+```
+
+- At the end of the file, don't return anything. The last line of the file should be the @notebook tag.
```

Comment:
I understand this is necessary and good now, but still makes me a bit sad that we're moving in the opposite direction from "single model == single result".
(not a problem)